src/main/java/com/playtika/homework1/task4/FirstAndSecondPositions.java

```diff
@@ -0,0 +1,56 @@
+package com.playtika.homework1.task4;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class FirstAndSecondPositions {
+
+    public static void main(String[] args) {
+        String[] names = {"Elena", "Thomas", "Hamilton", "Suzie", "Phil", "Matt", "Alex", "Emma", "John", "James", "Jane", "Emily", "Daniel", "Neda", "Aaron", "Kate"};
+        int[] times = {341, 273, 278, 329, 445, 402, 388, 275, 243, 334, 412, 393, 299, 343, 317, 265};
+        allResults(names, times);
+        certainPosition(names, times);
+    }
+
+    private static void allResults(String[] names, int[] times) {
+        System.out.println("Полный список результатов: \n");
+        int[] copy = times.clone();
+        Arrays.sort(copy);
+        for (int i = 0; i < times.length; i++) {
```

Comment:
`times.length` to a variable and reuse; the same way change everywhere in the class